packages/mobile/e2e/src/usecases/SecureSend.js

```diff
@@ -1,11 +1,19 @@
 import { enterPinUiIfNecessary, inputNumberKeypad, sleep } from '../utils/utils'
+import { dismissBanners } from '../utils/banners'
+var faker = require('faker')
```

Comment:
FWIW, in general eschew `var` usage in favor of `let` or `const`.

For module imports, it would be good to converge on using `imports` (or the `import faker = require('faker')` depending what faker exports).

Reply:
- [x] Fixed use of var in commit [458cd4f](https://github.com/celo-org/wallet/pull/260/commits/458cd4fdcb98574a3f3591e5f372d1e1a095fbe6)

Module imports would be good to have clarification / standards on.